lib/bundler/cli.rb

```diff
@@ -261,7 +261,19 @@ def show(gem_name = nil)
       Show.new(options, gem_name).run
     end
     # TODO: 2.0 remove `bundle show`
-    map %w[list] => "show"
+
+    if Bundler.feature_flag.list_command?
+      desc "list", "list all gems in the bundle"
```

Comment:
I think that it is good to capitalize the beginning of the description line.

```diff
- desc "list", "list all gems in the bundle"
+ desc "list", "List all gems in the bundle"
```

Related PR #5841.